sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/implementation/quickpulse/QuickPulseDataFetcherTests.java

```diff
@@ -105,11 +113,13 @@ void endpointChangesWithRedirectHeaderAndGetNewPingInterval() {
                 "instance1",
                 "machine1",
                 "qpid123",
-                "testSdkVersion");
+                "testSdkVersion",
+                quickPulseConfiguration);
         QuickPulseHeaderInfo quickPulseHeaderInfo = quickPulsePingSender.ping(null);
         assertThat(QuickPulseStatus.QP_IS_ON).isEqualTo(quickPulseHeaderInfo.getQuickPulseStatus());
         assertThat(1000).isEqualTo(quickPulseHeaderInfo.getQpsServicePollingInterval());
         assertThat("https://new.endpoint.com")
             .isEqualTo(quickPulseHeaderInfo.getQpsServiceEndpointRedirect());
+        assertThat(quickPulseConfiguration.getEtag()).isEqualTo("0::randometag::1::");
```

Comment:
Similar to comment above, would check against empty string.

Reply:
fixed